pages/pages.json

```diff
@@ -1,34 +1,14 @@
 {
   "pages": {
     "carousel": {
-      "path": "/",
+      "path": "/test",
```

Comment:
Not sure about that, but I have no idea how it was supposed to be.

Reply:
👍

Reply:
Who knows @leticiawanderley ? @klzns ?